.circleci/config.yml

```diff
@@ -1,120 +1,116 @@
 version: 2.1
 
-# add your orb below, to be used in integration tests (note: a @dev:alpha
-# release must exist; you'll need to publish manually at least once)
-# you can use scripts/publish-alpha.sh to publish a @dev:alpha release
-orbs:
-  codacy: codacy/base@dev:alpha
-  orb-tools: circleci/orb-tools@8.5.0
+references:
+  git_job: &git_job
+    docker:
+      - image: codacy/git-version:1.0.8
+    working_directory: ~/workdir
+
+  circleci_job: &circleci_job
+    docker:
+      - image: circleci/circleci-cli:0.1.5705
```

Comment:
Do you think i should use latest here too? I'm afraid they might break something without warning.

Reply:
yep, keep latest only for things we own and control the release.